.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -22,11 +25,14 @@
 Screenshots or GIFs are preferred. -->
 
 ### PR Tasks
-<!-- These tasks must be completed before a PR is merged.
-Delete tasks if they are not applicable. -->
+<!-- 
+These tasks must be completed before a PR is merged.
+If a task does not apply, write [na] instead of checking the box.
```

Comment:
nit: above you used `n/a`. here you used `na`.

Reply:
Personal preference I guess. As long as PR submitters do something along these lines, it's fine.